extensions/swagger-ui/deployment/src/test/java/io/quarkus/swaggerui/deployment/SwaggerOptionsTest.java

```diff
@@ -31,7 +31,7 @@ public void customOptions() {
                         containsString("https://petstore.swagger.io/v2/swagger.json"),
                         containsString("theme-newspaper.css"),
                         containsString("docExpansion: 'full'"),
-                        containsString("oauth2RedirectUrl: '/somesecure/page/oauth.html'"),
+                        containsString("var oar = \"/somesecure/page/oauth.html\";"),
```

Comment:
I have no idea about this, but I trust it's correct :)

Reply:
Yes, because the oauth redirect is now appending the host and port when relative I needed some javascript so, the value that gets replaced by the java code is not in the `const` anymore, but in the function. Hope that make sense